webextension/background.js

```diff
@@ -72,3 +73,28 @@ function disableAddon(tabId) {
   browser.browserAction.disable(tabId);
   browser.browserAction.setTitle({ tabId, title: "Containers disabled in Private Browsing Mode" });
 }
+
+browser.tabs.onCreated.addListener(tab => {
```

Comment:
Makes more sense to contain `browser.*` API calls when they may share some code and/or scope. Like the `themeManager`. So something more like ...
```
const TabPageCountManager = {
  init() {
    this.counter = {};
    browser.tabs.onCreated.addListener(this.initTabPageloadCounter);
    browser.tabs.onRemoved.addListener(this.sendTabPageloadCountAndDelete);
    browser.webRequest.onCompleted.addListener(this.incrementTabPageloadCount, {urls: ["<all_urls>"], types: ["main_frame"]});
  },
  initTabPageloadCounter() { ... },
  sendTabPageloadCountAndDelete() { ... },
  incrementTabPageloadCount() { ... }
}
```